haxe/content.md

```diff
@@ -0,0 +1,63 @@
+# What is Haxe?
+
+[Haxe](https://haxe.org) is an open source toolkit based on a modern, high level, strictly typed programming language, a cross-compiler, a complete cross-platform standard library and ways to access each platform's native capabilities.
+
+The Haxe compiler can output a number of source and binary files. As of Haxe 3.4.0-rc.1, the Haxe compiler can target JavaScript, Java, C#, C++, Python, PHP, Flash SWF, ActionScript 3, Lua, and Neko.
+
+%%LOGO%%
+
+# About this image
+
+This image ships a minimal Haxe toolkit:
+
+-	the `haxe` compiler with its standard library
+-	the `haxelib` library manager
+-	the `neko` virtual machine
+
+# How to use this image
+
+The most straightforward way to use this image is to use a Haxe container as both the build and runtime environment. In your `Dockerfile`, writing something along the lines of the following will compile and run your project:
+
+```dockerfile
+FROM haxe:3.4
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
The `RUN mkdir -p ...` here is no longer necessary (much older versions of Docker used to have issues with that sort of thing, but they've been fixed). :+1: